### ISSUE TITLE
XYZ sensitivity

### ISSUE THREAD

**Issue by nataelj** [ncdcommunity/Arduino_Library_ITG3200_16Bit_3Axis_Gyro_Angular_Sensor#2]
I think your x, y, and z values need to be divided by a GYRO_SENSITIVITY = 14.375 LSB per °/s. Other drivers for this device all seem to use this and it's mentioned in the docs; as best I can tell that's all that separates your current state from outputting values in actual degrees.

Though I'm just learning about all this so I may be misunderstanding something. :-)

**Comment by vbshightime** [comment]
Yes, you are correct, The 2000 degree/sec is the full-scale resolution of the device to get the exact value of the axis of rotation you need to divide your raw values with the scale factor defined in the following [datasheet](https://www.invensense.com/wp-content/uploads/2015/02/ITG-3200-Datasheet.pdf) in Gyroscope sensitivity column. This is the (sensitivity/scale) factor.  If you have chosen 2000 deg/sec then you need to divide it by 14.375 LSB/(º/s) to get the exact value.
